pandas-stubs/core/series.pyi

```diff
@@ -1743,6 +1743,8 @@ class TimestampSeries(Series[Timestamp]):
     # ignore needed because of mypy
     @property
     def dt(self) -> TimestampProperties: ...  # type: ignore[override]
+    @overload
+    def __add__(self, other: Timedelta | np.timedelta64) -> TimedeltaSeries: ...
```

Comment:
result here of adding `TimestampSeries` plus a timedelta is `TimestampSeries`

Reply:
sir here there is no `TimestampSeries`

Reply:
Look at line 1742